MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -144,6 +144,13 @@ open class RouteProgress: Codable {
         return currentLegProgress.upcomingStep ?? upcomingLeg?.steps.first
     }
     
+    /**
+     Upcoming `RouteAlerts` as reported by the navigation engine.
+     
+     Contents of the array depends on user's current progress along the route and are modified on each location update. You can use this to get info about incoming POIs.
+     */
+    public internal(set) var upcomingRouteAlerts: [UpcomingRouteAlertInfo] = []
```

Comment:
Are we able to provide any information on how far ahead "upcoming" is supposed to be? Is it tunable or related to speed of travel or current step?

Reply:
Good question. At least I am not aware of such configurations, and this is decided by NN. I'll investigate

Reply:
So I found the doc should be a bit corrected: `upcomingRouteAlerts` contain all events, that user still didn't pass entirely. In other words, at the beginning of the route this array holds all events from all the route. Event is removed from this array only when user has passed it's `end coordinate`